CHANGELOG.md

```diff
@@ -34,6 +34,9 @@
   capability checks that Firecracker performs during boot. If any of
   these fields are in use, minimal target snapshot version is
   restricted to 1.5.
+- [#4145](https://github.com/firecracker-microvm/firecracker/pull/4145):
+  Added support for per net device metrics. Now, along with aggregated
```

Comment:
nit: "net" and "network" are both used. Could we unify?

Reply:
this was updated in the later commits.